src/renderable/sprite.js

```diff
@@ -69,17 +69,20 @@
             // a flag to reset animation
             this.resetAnim = null;
 
+            // name of current frame
+            this.currentAnimName = null;
```

Comment:
should this one not set to "default" or something like this in case there is no animation defined (like just a fix sprite) ? Based on the current code, `setRegion` would certainly fail if you try to change the current region of a defined fixed sprite

Reply:
if you could as well add a test case for it in the corresponding test unit, that would be awesome, just to be sure we cover all possible use case with me.Sprite and ensure no regressions.

Reply:
I assumed by defined fixed sprite you mean a sprite object that has only one frame/no sprite-sheet. As such, I am adding a check in `setRegion`, if the `this.currentAnimName` is null, I just add a dummy animation named `default` and set `this.currentAnimName` to the default str.